erpnext/stock/get_item_details.py

```diff
@@ -165,6 +165,7 @@ def get_basic_details(args, item):
 		"discount_percentage": 0.0,
 		"supplier": item.default_supplier,
 		"delivered_by_supplier": item.delivered_by_supplier,
+		"actual_qty": get_item_balance_qty_for_default_warehouse(item)
```

Comment:
This need not always be default warehouse, since you are allowed to change the warehouse. Also, use the existing function: `erpnext.stock.utils.get_stock_balance`.